src/matter/cluster/TemperatureMeasurementCluster.ts

```diff
@@ -20,7 +20,7 @@ export const TemperatureMeasurementCluster = Cluster({
     /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.3.4 */
     attributes: {
        /** MeasuredValue = 100 x temperature [°C] **/
-       measuredValue: Attribute(0, TlvNullable(TlvInt16)),
+       measuredValue: Attribute(0, TlvNullable(TlvInt16.bound({ min: -27315, max: 32767 }))),
```

Comment:
32767 is already the maximum for an TlvInt16 so no need to bound it again (but min is lower)